lib/src/patterns/observer/Observer.dart

```diff
@@ -25,18 +25,16 @@ part of puremvc;
  *
  * See [IView], [INotification]
  */
-class Observer implements IObserver
-{
-
+class Observer implements IObserver {
   /**
    * This [IObserver]'s [notifyMethod] (i.e., callback)
    */
-  Function notifyMethod;
+  Function? _notifyMethod;
```

Comment:
remove the underscores throughout this class

Reply:
Access modifier added.

Reply:
Access modifiers were added, because they are present in interface (and Dart does not allow direct mapping from property to access modifiers as it was before)